packages/components/package.json

```diff
@@ -23,9 +23,7 @@
     "./utilities/*": "./dist/utilities/*",
     "./translations/*": "./dist/translations/*"
   },
-  "files": [
-    "dist"
-  ],
+  "files": ["dist"],
```

Comment:
The beautifier again :). Will make sure to tackle #181 in the next couple of days :)